fatips/2.md

```diff
@@ -0,0 +1,264 @@
+| `from`     | string  | The pegged asset token ID to convert from                    | Must be a valid pegged token symbol                 | Y        |
+| `input`    | string  | The Public Factoid address to us as an input for `from` type of pegged token | Must be a valid public Factoid address              | Y        |
+| `to`       | string  | The pegged asset token ID to convert to from `from`          | Must be a valid pegged token symbol                 | Y        |
+| `output`   | string  | The Public Factoid address to us as an output for `to` type of pegged token | Must be a valid public Factoid address              | Y        |
+| `amount`   | number  | The amount of `from` to use as an input to the conversion    | Must be greater than `0`                            | Y        |
+| `origin`   | boolean | Whether to apply `amount` as the input amount of the conversion. `amount` is applied as the conversion output amount of `to` by default. | Valid boolean. May be omitted for default behavior. | N        |
+| `metadata` | any     | Arbitrary user defined metadata for the conversion           | Must be valid JSON                                  | N        |
+
+##### Signing
+
+Conversions are signed according to [FATIP-103](103.md). The signing set is
+the key corresponding to `input` field. The conversion must include an RCD/Signature pair for the source address of the conversion. Signatures and RCDs are defined in the External IDs of the conversion entry as laid out in FATIP-103.
+
+#### Conversion Transaction Validation
+
+Conversions must meet all of the following criteria to be valid.
+
+General Criteria:
+
+- The content of the entry must be a single well-formed JSON.
+- The JSON must contain all required fields listed in the above table, all fields and their members must be of the correct type. No unspecified fields may be present. No duplicate field names are allowed.
+- The entry hash of the conversion entry must be unique among all
+  previously valid conversions of the pegged asset token.
+- The External IDs must follow the cryptographic and structural
+  requirements defined by [FATIP-103](103.md) for the `address` input
+
+Specific Criteria:
+
+- If `origin` is true, `input` must own at least `amount` of pegged token `from` for the conversion to be valid
+- If `origin` is false or omitted, `input` must own at least `amount` of pegged token `from` at current ORP exchange rate for the conversion to be valid
+
+
+
+If all criteria is met, the following occurs as a single atomic operation:
+
+- `amount` (or equivalent at current OPR exchange rate if `origin` is true) of token `from` is burnt (destroyed) in address `input`
+- `from` is converted to PNT (interim) at the OPR exchange rate
+- Interim PNT is expended to create `to`  via coinbase transaction to `output` at the current OPR exchange rate
+
+The conversion is complete.
```

Comment:
why is "pnt" used as interim? all the values in the opr are expressed as relative to USD, so we should use USD as the base

let's say we have X pFCT and want to convert it to pEUR::
```
X pFCT = X / pFCT_Rate pUSD
       = X / pFCT_Rate * PNT_Rate PNT
       = X / pFCT_Rate * PNT_Rate pUSD
       = X / pFCT_Rate * PNT_Rate / PNT_Rate * pEUR_Rate pEUR```
       

as you can see you're multiplying and then immediately dividing by PNT_Rate, meaning it's an entirely unnecessary step when we can just do `X pFCT = ((X/pFCT_Rate) * pEUR_Rate) pEUR` where X/pFCT_rate is pUSD